Adapter/Adapter_HookManager.php

```diff
@@ -24,10 +24,16 @@
  *  International Registered Trademark & Property of PrestaShop SA
  */
 
+use PrestaShop\PrestaShop\Core\Business\Dispatcher\HookEvent;
+
+/**
+ * This adapter will be registered as a EventDispatcher listener, on the 'hook' dispatcher,
+ * and will allow legacy modules/legacy hooks to be called by the new Architecture.
+ */
 class Adapter_HookManager
```

Comment:
Just being picky here, but [don't call your class `Manager`](http://blog.codinghorror.com/i-shall-call-it-somethingmanager/) if you can :)
This seems to build a bridge between the event system and the hooks system, so why not call it something like `Adapter_EventToHookBridge`?